<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Frame Section Props</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../civiltools.qrc">
    <normaloff>:/civiltools/images/property_editor.svg</normaloff>:/civiltools/images/property_editor.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="concrete_groupbox">
     <property name="title">
      <string>Concrete</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Concrete</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="concrete_mats">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Suffix Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="concrete_suffix">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="clean_names_checkbox">
        <property name="text">
         <string>Clean Names</string>
        </property>
        <property name="icon">
         <iconset resource="../../civiltools.qrc">
          <normaloff>:/civiltools/images/edit-clear.svg</normaloff>:/civiltools/images/edit-clear.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="columns">
       <property name="text">
        <string>Columns</string>
       </property>
       <property name="icon">
        <iconset resource="../../civiltools.qrc">
         <normaloff>:/civiltools/images/columns.svg</normaloff>:/civiltools/images/columns.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="beams">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Beams</string>
       </property>
       <property name="icon">
        <iconset resource="../../civiltools.qrc">
         <normaloff>:/civiltools/images/beams.svg</normaloff>:/civiltools/images/beams.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="stories">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="selection_checkbox">
     <property name="text">
      <string>Selections</string>
     </property>
     <property name="icon">
      <iconset resource="../../civiltools.qrc">
       <normaloff>:/civiltools/images/select-area-icon.svg</normaloff>:/civiltools/images/select-area-icon.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="assign_pushbutton">
     <property name="text">
      <string>Apply</string>
     </property>
     <property name="icon">
      <iconset resource="../../civiltools.qrc">
       <normaloff>:/civiltools/images/etabs.png</normaloff>:/civiltools/images/etabs.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../civiltools.qrc"/>
 </resources>
 <connections/>
</ui>
